Assets/MRTK/Tests/EditModeTests/Core/Extensions/CameraExtensionTests.cs

```diff
@@ -154,7 +154,10 @@ public void TestIsInFOV()
 
             var far = testCamera.farClipPlane;
             var frustrumSize = testCamera.GetFrustumSizeForDistance(far / 2.0f);
-            Assert.IsTrue(testCamera.IsInFOV(new Vector3(frustrumSize.x / 2.0f, frustrumSize.y / 2.0f, far / 2.0f)));
+            /// (fSz.x,fSz.y,far) is right on the upper right corner midway up the frustum. Numerical precision can
+            /// put it either in or out of frustum. Subtract a small epsilon to bring it inside.
+            float eps = 1.0e-5f;
```

Comment:
@keveleigh pointed this out to me in some other places, but there's https://docs.microsoft.com/en-us/dotnet/api/system.double.epsilon?view=netcore-3.1 which can be neat.

Reply:
There's also a Unity-defined epsilon! https://docs.unity3d.com/ScriptReference/Mathf.Epsilon.html

Reply:
Agreed in general on preferring built in epsilons, but this one had to be hand tuned to get the tests to pass reliably. I'm hoping even Unity's epsilon is smaller than 1e-5.